src/components/HealthClimateComponents/GroundLevel.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  grid: {
+    backgroundColor: '#F3F3F3',
+    height: '45rem'
+  },
+  mainTitle: {
+    color: theme.palette.primary.light,
+    textAlign: 'center',
+    fontWeight: 'bolder',
+    textDecoration: 'Upppercase',
```

Comment:
Only include styles that are not _default_ and not defined in `App.js` theme. I believe `uppercase`, etc. are already defined there.